source/OutfitterPanel.cpp

```diff
@@ -1036,3 +1222,177 @@ int OutfitterPanel::FindItem(const string &text) const
+
+	// Clear the buttonZones, they will be populated again as buttons are drawn.
+	buttonZones.clear();
+
+	// Row 1.
+	DrawButton("_Buy", Rectangle(Point(buttonCenterX + buttonOffsetX * -1, rowBaseY + rowOffsetY * 0), buttonSize),
+		ButtonActive('b'), hoverButton == 'b', 'b');
+	DrawButton("_Install", Rectangle(Point(buttonCenterX + buttonOffsetX * 0, rowBaseY + rowOffsetY * 0), buttonSize),
+		ButtonActive('i'), hoverButton == 'i', 'i');
+	DrawButton("_Cargo", Rectangle(Point(buttonCenterX + buttonOffsetX * 1, rowBaseY + rowOffsetY * 0), buttonSize),
+		ButtonActive('c'), hoverButton == 'c', 'c');
+	// Row 2.
+	DrawButton("_Sell", Rectangle(Point(buttonCenterX + buttonOffsetX * -1, rowBaseY + rowOffsetY * 1), buttonSize),
+		ButtonActive('s'), hoverButton == 's', 's');
+	DrawButton(!CanMoveOutfit(OutfitLocation::Ship, OutfitLocation::Storage) &&
+		CanMoveOutfit(OutfitLocation::Cargo, OutfitLocation::Storage) ? "_Unload" : "_Uninstall",
+		Rectangle(Point(buttonCenterX + buttonOffsetX * 0, rowBaseY + rowOffsetY * 1), buttonSize),
+		ButtonActive('u'), hoverButton == 'u', 'u');
+	DrawButton("Sto_re", Rectangle(Point(buttonCenterX + buttonOffsetX * 1, rowBaseY + rowOffsetY * 1), buttonSize),
+		ButtonActive('r'), hoverButton == 'r', 'r');
+	// Row 3.
+	DrawButton("_Leave", Rectangle(Point(buttonCenterX + buttonOffsetX * 1, rowBaseY + rowOffsetY * 2), buttonSize),
+		true, hoverButton == 'l', 'l');
+
+	// Draw the Modifier hover text that appears below the buttons when a modifier is being applied.
+	int modifier = Modifier();
+	if(modifier > 1)
+	{
+		string mod = "x " + to_string(modifier);
+		int modWidth = font.Width(mod);
+		for(int i = -1; i < 2; i++)
+			font.Draw(mod, Point(buttonCenterX + buttonOffsetX * i, rowBaseY + rowOffsetY * 0)
+			+ Point(-.5 * modWidth, 10.), dim);
+		for(int i = -1; i < 2; i++)
+			font.Draw(mod, Point(buttonCenterX + buttonOffsetX * i, rowBaseY + rowOffsetY * 1)
+			+ Point(-.5 * modWidth, 10.), dim);
+	}
+
+	// Draw tooltips for the button being hovered over:
+	string tooltip = GameData::Tooltip(string("outfitter: ") + hoverButton);
+	if(!tooltip.empty())
+		buttonsTooltip.IncrementCount();
+	else
+		buttonsTooltip.DecrementCount();
+
+	if(buttonsTooltip.ShouldDraw())
+	{
+		buttonsTooltip.SetZone(buttonsFooter);
+		buttonsTooltip.SetText(tooltip, true);
+		buttonsTooltip.Draw();
+	}
+
+	// Draw the tooltip for your full number of credits and free cargo space
+	const Rectangle creditsBox = Rectangle::FromCorner(creditsPoint, Point(SIDEBAR_WIDTH - 20, 30));
+	if(creditsBox.Contains(hoverPoint))
+		creditsTooltip.IncrementCount();
+	else
+		creditsTooltip.DecrementCount();
+
+	if(creditsTooltip.ShouldDraw())
+	{
+		creditsTooltip.SetZone(creditsBox);
+		creditsTooltip.SetText(Format::Number(player.Accounts().Credits()) + " credits" + "\n" +
+			Format::Number(player.Cargo().Free()) + " tons free out of " +
+			Format::Number(player.Cargo().Size()) + " tons total capacity", true);
```

Comment:
```suggestion
			Format::MassString(player.Cargo().Free()) + " free out of " +
			Format::MassString(player.Cargo().Size()) + " total capacity", true);
```
For proper pluralization.